cli/config.go

```diff
@@ -23,108 +23,110 @@
 package cli
 
 import (
+	"errors"
 	"fmt"
+	"strings"
 
 	"github.com/urfave/cli/v2"
 
-	"github.com/temporalio/tctl-kit/pkg/color"
 	"github.com/temporalio/tctl-kit/pkg/config"
 )
 
 func newConfigCommands() []*cli.Command {
 	return append([]*cli.Command{
 		{
-			Name:  "get",
-			Usage: "Print config values",
-			Flags: []cli.Flag{},
+			Name:      "get",
+			Usage:     "Print the value of an env property",
+			Flags:     []cli.Flag{},
+			ArgsUsage: "env.[ENV NAME].[PROPERTY NAME] or [PROPERTY NAME] for current env",
 			Action: func(c *cli.Context) error {
-				return GetValue(c)
+				return EnvProperty(c)
 			},
 		},
 		{
-			Name:  "set",
-			Usage: "Set config values",
-			Flags: []cli.Flag{},
-
+			Name:      "set",
+			Usage:     "Set the value of an env property",
+			Flags:     []cli.Flag{},
+			ArgsUsage: `env.[ENV NAME].[PROPERTY NAME] [VALUE] or [PROPERTY NAME] [VALUE] for current env`,
 			Action: func(c *cli.Context) error {
-				return SetValue(c)
+				return SetEnvProperty(c)
 			},
 		},
 	}, newEnvCommands()...)
 }
 
-func GetValue(c *cli.Context) error {
-	for i := 0; i < c.Args().Len(); i++ {
-		key := c.Args().Get(i)
-		val, err := tctlConfig.Get(key)
+func EnvProperty(c *cli.Context) error {
+	if c.NArg() != 1 {
+		return errors.New("invalid number of args, expected 1: env property name")
+	}
 
-		if err != nil {
-			return err
-		}
+	fullKey := c.Args().Get(0)
 
-		fmt.Printf("%v: %v\n", color.Magenta(c, "%v", key), val)
+	if err := validateEnvKey(fullKey); err != nil {
+		return err
 	}
 
+	env, key := envKey(fullKey)
+
+	val := tctlConfig.EnvProperty(env, key)
+	fmt.Println(val)
+
 	return nil
 }
 
-func SetValue(c *cli.Context) error {
-	if c.Args().Len() == 0 {
-		return fmt.Errorf("no key specified")
+func SetEnvProperty(c *cli.Context) error {
+	if c.NArg() != 2 {
+		return errors.New("invalid number of args, expected 2: property and value")
 	}
 
-	key := c.Args().Get(0)
+	fullKey := c.Args().Get(0)
+	val := c.Args().Get(1)
 
-	var val string
-	if c.Args().Len() > 1 {
-		val = c.Args().Get(1)
+	if err := validateEnvKey(fullKey); err != nil {
+		return err
 	}
 
-	if err := tctlConfig.Set(key, val); err != nil {
-		return fmt.Errorf("unable to set property %s: %s", key, err)
+	if fullKey == "version" {
+		if err := tctlConfig.SetVersion(val); err != nil {
+			return fmt.Errorf("unable to set version: %s", err)
+		}
+
+		return nil
 	}
 
-	fmt.Printf("%v: %v\n", color.Magenta(c, "%v", key), val)
+	env, key := envKey(fullKey)
+
+	if err := tctlConfig.SetEnvProperty(env, key, val); err != nil {
+		return fmt.Errorf("unable to set env property %v. %s", key, err)
+	}
 
+	fmt.Printf("%v: %v\n", key, val)
```

Comment:
We should print the full key here to let user understand which env the key was set for.

```suggestion
	fmt.Printf("Set '%v' to: %v\n", fullKey, val)
```